cmds/wifi/wifi.go

```diff
@@ -30,6 +32,14 @@ const (
 	}`
 )
 
+var (
+	CellRE       = regexp.MustCompile("(?m)^\\s*Cell")
+	EssidRE      = regexp.MustCompile("(?m)^\\s*ESSID.*")
+	EncKeyOptRE  = regexp.MustCompile("(?m)^\\s*Encryption key:(on|off)$")
```

Comment:
one way to make the REs a bit more resilient to changes in the tool is to replace all spaces with  *, e.g.
IE: IEEE 802.11i/WPA2 Version 1
IE: *IEEE *802.11i/WPA2 *Version 1

but this is a stretch, I'm sure it's fine.